runtime/lua/vim/_defaults.lua

```diff
@@ -225,13 +225,13 @@ do
     })
 
     vim.keymap.set('n', '[Q', function()
-      vim.cmd.crewind({ args = vim.v.count ~= 0 and { vim.v.count } or nil })
+      vim.cmd.crewind({ count = vim.v.count ~= 0 and vim.v.count or nil })
```

Comment:
`:h :crewind` says:

```
:cr[ewind][!] [nr]	Display error [nr].  If [nr] is omitted, the FIRST
			error is displayed.  See |:cc|.
```

so why does `args` not work here? `:crewind 1` works as expected

Reply:
`count` has the same effect (this isn't obvious to me from the help description) so this is fine.

Reply:
`vim.api.nvim_parse_cmd("crewind 123", {})`  do actually parse the argument to `count` 

```
{                                                                                    
  addr = "?",                                                                                  
  args = {},                                                                                   
  bang = false,                                                                                
  cmd = "crewind",                                                                             
  count = 123,                                                                                 
  magic = {                                                                                    
    bar = true,                                                                                
    file = false                                                                               
  },                                                                                           
  mods = {                                                                                     
    browse = false,                                                                            
    confirm = false,                                                                           
    emsg_silent = false,                                                                       
    filter = {                                                                                 
      force = false,                                                                           
      pattern = ""                                                                             
    },                                                                                         
    hide = false,                                                                              
    horizontal = false,                                                                        
    keepalt = false,                                                                           
    keepjumps = false,                                                                         
    keepmarks = false,                                                                         
    keeppatterns = false,                                                                      
    lockmarks = false,                                                                         
    noautocmd = false,                                                                         
    noswapfile = false,                                                                        
    sandbox = false,                                                                           
    silent = false,                                                                            
    split = "",                                                                                
    tab = -1,                                                                                  
    unsilent = false,                                                                          
    verbose = -1,                                                                              
    vertical = false                                                                           
  },                                                                                           
  nargs = "0",                                                                                 
  nextcmd = "",                                                                                
  range = { 123 }                                                                              
}   
```

Reply:
It was #23121